### PR TITLE
Add libimobiledevice / update iproxy (@NVISOSecurity)

### PR DESCRIPTION
**Pull request by TheDauntless**
Added new tool page for libimobiledevice with an overview of all the available tools.

iproxy currently still has its own page (TOOL-0055) because it was already there. I don't mind having a general page for the libimobiledevice suite and splitting off a tool if useful, but let me know what you think.

### CODE REVIEW

File: tools/ios/MASTG-TOOL-0126.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: libimobiledevice suite
+platform: ios
+host:
+- macOS
+- windows
+- linux
+source: https://libimobiledevice.org/
+---
+
+The libimobiledevice suite is cross-platform protocol library for interacting with iOS devices. The different libraries can be compiled into binaries for direct interaction with iOS devices from the commandline.
```
Comment (sushi2k) [inline tools/ios/MASTG-TOOL-0126.md:11]:
```suggestion
The libimobiledevice suite is a cross-platform protocol library for interacting with iOS devices. The different libraries can be compiled into binaries for direct interaction with iOS devices from the command line.
```

Comment (sushi2k) [reply tools/ios/MASTG-TOOL-0126.md:11]: Is it `commandline` or `command line`?

**Review by sushi2k — COMMENTED** [review]
Thank you for the update @TheDauntless! One small thing to check.

**Comment by sushi2k** [comment]
All of the changes in here are also covered in #3100. Shall we close this PR @TheDauntless?

**Comment by TheDauntless** [comment]
Closed, as these changes are part of #3100

**Pull request closed without merging**